1-JavaScript/Week2/homework/ex2-dogYears.js

```diff
@@ -16,14 +16,15 @@ calculate it!
    ages.
 -----------------------------------------------------------------------------*/
 
-function calculateDogAge(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function calculateDogAge(puppyAge) {
+  const ageInDogYear=puppyAge*7;
```

Comment:
Also here you need some more space 🙂 

```suggestion
  const ageInDogYear = puppyAge * 7;
```